pkg/ddevapp/ddevapp.go

```diff
@@ -306,7 +307,9 @@ func (app *DdevApp) Describe(short bool) (map[string]interface{}, error) {
 		services[shortName]["short_name"] = shortName
 		var ports []string
 		for pk := range c.Config.ExposedPorts {
-			ports = append(ports, pk.Port())
+			if !slices.Contains(ports, pk.Port()) {
+				ports = append(ports, pk.Port())
+			}
```

Comment:
I was wondering why `InDocker: solr:8983,8983`, the same port is listed twice.
That's where it comes from.